watson-developer-cloud/discovery-components
# fix: upgrade pdfjs to 2.6.347

#### What do these changes do/fix?

- upgrade pdfjs-dist version in package.json
- update types according to new types system (internally build rather than DefinitelyTyped)

Contributes to https://github.ibm.com/Watson-Discovery/disco-issue-tracker/issues/12076

#### How do you test/verify these changes?
- Run storybook and check that PDF stories work correctly.
- Link with tooling and ensure that PDFs render correctly. Note that the linked issue is fixed.

#### Have you documented your changes (if necessary)?

#### Are there any breaking changes included in this pull request?
